scania-digital-design-system/tegel
# docs(storybook): prod and develop versions

**Describe pull-request**  
Adding links to develop and prod version of Storybook.

**Solving issue**  
Fixes: [CDEP-2331](https://tegel.atlassian.net/browse/CDEP-2331)

**How to test**  
1. Go to `CODE_STYLES` document, check info about develop version of Storybook.
2. Check out README's of `root` and `core`, check info about prod version of Storybook.

**Suggested test steps**
